### DISCUSSION TITLE
疑似 ClassIsland 的 icon 不对称

### DISCUSSION THREAD

**Discussion by LyCecilion** [ClassIsland/ClassIsland#231] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/2ed88163-9071-4bce-9581-05263dfbd511)

上面四个箭头是我在 Snipaste 随手画的，但是大概是等长的。一图胜千言。不知道是不是特殊的艺术处理。

**Comment by HelloWRC** [comment, marked as answer]
这个可能是画图标时没注意到（）
